src/main/java/pl/edu/pw/domain/TicketType.java

```diff
@@ -0,0 +1,28 @@
+package pl.edu.pw.domain;
+
+import java.util.Locale;
+
+public enum TicketType {
+
+    ADULT, STUDENT, CHILD;
+
+    public static double ticketCost(TicketType ticketType) {
```

Comment:
Ogólnie to enumy mogą mieć niestatyczne metody i wtedy odpalasz je tak: `TicketType.ADULT.ticketCost()` + parametry w konstruktorze enuma. Pomijając fakt, jak pisał Dominik - ta wartość nie wydaje się być cechą enumów ;)